diff --git a/src/duckdb/src/include/duckdb/common/enums/row_id_handling.hpp b/src/duckdb/src/include/duckdb/common/enums/row_id_handling.hpp
index 9533ce112..dd9f7fbe9 100644
--- a/src/duckdb/src/include/duckdb/common/enums/row_id_handling.hpp
+++ b/src/duckdb/src/include/duckdb/common/enums/row_id_handling.hpp
@@ -9,6 +9,7 @@
 #pragma once
 
 #include "duckdb/common/constants.hpp"
+#include "duckdb/common/windows_undefs.hpp" // test1
 
 namespace duckdb {
 
